.storybook/preview-head.html

```diff
@@ -1,8 +1,11 @@
 <link href="https://fonts.googleapis.com/css?family=Roboto:100,200,300,400,500" rel="stylesheet" />
-<link href="http://fonts.cdnfonts.com/css/sofia-pro" rel="stylesheet" />
-<link href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet">
-<link href='//fonts.googleapis.com/css2?family=Noto+Kufi+Arabic:wght@300;400;500;700&display=swap' rel='stylesheet'>
-<link href='https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300;400;500;700' rel='stylesheet'>
+<link href="https://fonts.cdnfonts.com/css/sofia-pro" rel="stylesheet" />
```

Comment:
changed http to https

Reply:
![image](https://user-images.githubusercontent.com/35146205/154285816-32dbf140-346c-4603-b05a-95ff68c8511b.png)